README.md

```diff
@@ -21,7 +21,8 @@
   - [Build multi-arch images on arch-specific self-hosted runners in parallel](#build-multi-arch-images-on-arch-specific-self-hosted-runners-in-parallel)
   - [Build images locally](#build-images-locally)
 - [Advanced Dependencies](#advanced-dependencies)
-  - [Extra System Dependencies (apt)](#extra-system-dependencies-apt)
+  - [Extra System Dependencies (*apt*)](#extra-system-dependencies-apt)
+  - [Extra System Dependencies (*pip*)](#extra-system-dependencies-pip)
   - [Custom Installation Script](#custom-installation-script)
   - [Extra Image Files](#extra-image-files)
 - [Configuration Variables](#configuration-variables)
```

Comment:
update version number to 1.1.0 in readme (and search for any other occurrences)

Reply:
Especially [this](https://github.com/ika-rwth-aachen/docker-ros/tree/improvement/naming-of-special-files#about) section needs to be updated